wger/exercises/management/commands/download-exercise-images.py

```diff
@@ -67,9 +67,12 @@ def handle(self, **options):
         except ValidationError:
             raise CommandError('Please enter a valid URL')
 
-        exercise_api = "{0}/api/v2/exercise/?limit=999"
+        #Adding 'status=2' at the end of the exercise_api url will filter the json data for only accepted exercises 
+        exercise_api = "{0}/api/v2/exercise/?limit=999;status=2"
```

Comment:
- Changes:

    Substituted the semicolon for an ampersand at the "exercise_api" variable.

I think that both can be used as query separators since I was getting the same results with the semicolon and ampersand. 

https://wger.de/api/v2/exercise/?limit=999;status=2
https://wger.de/api/v2/exercise/?limit=999&status=2